log/config.go

```diff
@@ -0,0 +1,29 @@
+package log
+
+type typeKey int
+
+const Formatter typeKey = iota
+
+type Config interface {
+	TypeKey() interface{}
+	Apply(logger Logger) error
+}
+
+type standardFormat struct{}
+type jsonFormat struct{}
+
+func NewStandardFormat() Config             { return standardFormat{} }
+func (standardFormat) TypeKey() interface{} { return Formatter }
+func (sf standardFormat) Apply(logger Logger) error {
+	return applyFormatter(sf, logger)
+}
+
+func NewJSONFormat() Config             { return jsonFormat{} }
+func (jsonFormat) TypeKey() interface{} { return Formatter }
+func (jf jsonFormat) Apply(logger Logger) error {
+	return applyFormatter(jf, logger)
+}
+
+func applyFormatter(formatter Config, logger Logger) error {
+	return logger.(formattable).SetFormatter(formatter)
```

Comment:
nit: Having a function for this is overkill.

Reply:
If we more format in the future, it might be handy, and it reduces duplicates.